release-notes/release-6.16.0.md

```diff
@@ -0,0 +1,39 @@
+## Release summary
+
+Eclipse Che 6.16 includes:
+
+* **Update of the `/activity` REST endpoint**: which is now capable of returning information about
+workspace activitity over time.
+
+## Upgrading
+
+Instructions on how to upgrade.
+
+
+## Release details
+
+### Update of the `/activity` REST endpoint (https://github.com/eclipse/che/pull/12177)
```

Comment:
Change title to "Improved REST API endpoint /activity"

In the description: What does "a certain state for a certain time" mean? How would a user use this?

I *think* this means "You can add optional arguments to the /activity REST endpoint to query workspace IDs which are in a specific state (STARTING, RUNNING, or STOPPING) for longer than a given time." - it would be great to give a sample of how this might be used, and how it impacts the end user or admin.

Thanks!

Reply:
>  What does "a certain state for a certain time" mean? How would a user use this?

I wanted to be brief in the release notes. The generated Swagger documentation for the endpoint should contain more details. See https://github.com/eclipse/che/blob/master/wsmaster/che-core-api-workspace-activity/src/main/java/org/eclipse/che/api/workspace/activity/WorkspaceActivityService.java#L91 for the `@ApiOperation` and `@ApiParam` annotations.